recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "2.6.0" %}
+{% set version = "2.7.0" %}
 
 package:
   name: keras
   version: {{ version }}
 
 source:
-  url: https://files.pythonhosted.org/packages/5a/38/04d9b7fb53acdf861df2c4505fa96b06c779817a511e94b8882d284ba360/keras-{{ version }}-py2.py3-none-any.whl
-  sha256: 504af5656a9829fe803ce48a8580ef16916e89906aceddad9e098614269437e7
+  url: https://files.pythonhosted.org/packages/6b/8b/065f94ba03282fa41b2d76942b87a180a9913312c4611ea7d6508fbbc114/keras-2.7.0-py2.py3-none-any.whl
```

Comment:
Did we hardcode version in the URL due to any issue?

Reply:
It was hard-coded for some debugging purpose when keras 2.7 was not out. I'll revert that.